pages/clustering/high-availability.mdx

```diff
@@ -416,7 +416,7 @@ are all disabled.
 
 <Callout type="warning">
 
-Under certain extreme scenarios, the current implementation of HA could lead to data loss. These are environments with high volume of transactions 
+Under certain extreme scenarios, the current implementation of HA could lead to having Recovery Point Objective (RPO) != 0. These are environments with high volume of transactions 
```

Comment:
I know that this is a standard way, `RPO!=0`, but I suggest replacing notation with words. Something along the lines of: 

`Under certain extreme scenarios, the current implementation of High Availability (HA) could result in a Recovery Point Objective (RPO) greater than zero, meaning some data loss could occur.`